instrumentctl/g9_driver.py

```diff
@@ -0,0 +1,255 @@
+import serial
+import threading
+import time
+# from utils import LogLevel
+import os
+# from subsystem import interlocks
+
+inStatus = {
+    0: "No error",
+    1: "Invalid configuration",
+    2: 'External test signal failure',
+    3: 'Internal circuit error',
+    4: 'Discrepancy error',
+    5: 'Failure of the associated dual-channel input'
+}
+
+outStatus = {
+    0: 'No error',
+    1: 'Invalid configuration',
+    2: 'Overcurrent detection',
+    3: 'Short circuit detection',
+    4: 'Stuck-at-high detection',
+    5: 'Failure of the associated dual-channel output',
+    6: 'Internal circuit error',
+    8: 'Dual channel violation'
+}
+
+usStatus = {
+    9: "Output Power Supply Error Flag",
+    10: "Safety I/O Terminal Error Flag",
+    13: "Function Block Error Flag"
+}
+
+NUMIN = 13
+
+SNDHEADER = b'\x40\x00\x00\x0F\x4B\x03\x4D\x00\x01' 
+SNDDATA = b'\x00\x00\x00\x00'
+SNDRES = b'\x00\x00'
+
+RECHEADER = b'\x40\x00\x00'
+
+FOOTER = b'\x2A\x0D'
```

Comment:
these constants are specific to the G9Driver and won't be used or imported elsewhere, so I'd like these to be encapsulated within the class. Also these should be in all caps like:
```
class G9Driver:
    IN_STATUS = {
        0: "No error",
        1: "Invalid configuration",
        2: 'External test signal failure',
        3: 'Internal circuit error',
        4: 'Discrepancy error',
        5: 'Failure of the associated dual-channel input'
    }
```